### PR TITLE
Recurring events

### PR DESCRIPTION
**Pull request by isha-san**
Allow CAdmins to create recurring events. Docs here: https://docs.google.com/document/d/1PF6AAtswPTY7CMkDAf-DKUOcBaU1J9e-6wYO5aI6h8g/edit?usp=sharing

### CODE REVIEW

File: v2.0/src/Components/Pages/EventsPage/EventsPageReal.js [hunk 1]
```diff
@@ -178,23 +183,73 @@ class EventsPage extends React.Component {
...
     if (events) {
-      return events.map((event) => {
+      const page = events.map((event) => {
+        let recurringDetailString = "";
         const dateString = dateFormatString(
           new Date(event.start_date_and_time),
           new Date(event.end_date_and_time)
         );
-
+        if (event.is_recurring) {
```
Comment (frimpongopoku) [inline v2.0/src/Components/Pages/EventsPage/EventsPageReal.js:192]: Scary scary stuff.
Hi @isha-san , I know you have good intentions ,  but are we sure this is the only way to implement the recurring events...? Making api calls in the middle of rendering?

**Comment by frimpongopoku** [comment]
Hi @isha-san , when you get time, please resolve the warnings in the terminal. You can add this to your cleanup routine before you make PRs 🤗 
Screenshot 2021-07-19 at 10 11 06

**Review by BradHN1 — APPROVED** [review]
Looks good

**Comment by BradHN1** [comment]
I didn't notice this PR was merging to master.  PRs should merge to development, test in dev before merging to master

**Pull request merged**